bHY6yh-d12559/cod-yyyiou_bHY6yh-d12559.xml

```diff
@@ -0,0 +1,335 @@
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e215">Ibidem
+          <lb ed="#C"/>capitulo V</ref> <quote xml:id="bHY6yh-d12559-Qd1e208">quid prohibet dicere felicem secundum uirtu<lb ed="#C" break="no"/>tem 
+          perfectam operantem et extioribus bonis
+          <lb ed="#C"/>sufficienter ditatum</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e217">
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e228">Ibidem capitulo 8</ref> <quote xml:id="bHY6yh-d12559-Qd1e219">feli<lb ed="#C" break="no"/>citas 
+          est operatio quedam secundum uirtutem perfec<lb ed="#C" break="no"/>tam</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e225">
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e240">Ibidem libro VII capitulo XIIII</ref> <quote xml:id="bHY6yh-d12559-Qd1e229">Indiget
+          <lb ed="#C"/>felix hiis que in corpore bonis Et hiis ex<lb ed="#C" break="no"/>terius 
+          et fortuna ut impediat hoc</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e237">
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e252">Ibidem
+          <lb ed="#C"/>libro IX capitulo IX</ref> <quote xml:id="bHY6yh-d12559-Qd1e239">felix esse est bene uiuere et operari</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e245">
+          <lb ed="#C"/>¶ <ref xml:id="bHY6yh-d12559-Rd1e262">Ibidem libro X et finale capitulo XXVIII</ref> <quote xml:id="bHY6yh-d12559-Qd1e247">Uidetur fe<lb ed="#C" break="no"/>lix 
+          uita secundum uirtutem operatio uita felix
+          <lb ed="#C"/>est secundum uirtutem operantis videtur et an<lb ed="#C" break="no"/>axagore 
+          non diuitem neque potentem
+          <lb ed="#C"/>existimare felicem esse Est sapiens maxime
+          <lb ed="#C"/>felix</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e261">
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e280">Idem magnorum moralium libro I capitulo I</ref> <quote xml:id="bHY6yh-d12559-Qd1e263">Felicitate
+          <lb ed="#C"/>adueniente nullo alio indigemus felicitatem
+          <lb ed="#C"/>dicimus esse bene operari et bene uiuere bene uiuere
+          <lb ed="#C"/>et bene operari nihil aliud quam felicitatem dicimus
+          <lb ed="#C"/>ergo felicitas quid est bene uiuere In vsuqui<lb ed="#C" break="no"/>dem 
+          et actum est felicitas Felicitas est in uiuen<lb ed="#C" break="no"/>do 
+          secundum uirtutem optimum bonum est felici<lb ed="#C" break="no"/>tas 
+          et hec finis actu uiuentes secundum uirtu<lb ed="#C" break="no"/>tes 
+          felices erimus et habebimus perfectum bo<lb ed="#C" break="no"/>num 
+          finem</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e293">
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e314">Ibidem capitulo II</ref> <quote xml:id="bHY6yh-d12559-Qd1e295">perfectione uir<lb ed="#C" break="no"/>tutis 
+          actus est felicitas</quote>
+        </p>
+        <p xml:id="bHY6yh-d12559-d1e302">
+          ¶ <ref xml:id="bHY6yh-d12559-Rd1e325">Idem libro priomo
```

Comment:
is there a type here for "priomo". If not it should be placed in `<sic>` tag. Otherwise, please correct to "primo"
```suggestion
          ¶ <ref xml:id="bHY6yh-d12559-Rd1e325">Idem libro priomo
```